internal/core/application/ban.go

```diff
@@ -148,10 +148,12 @@ func (s *service) banSignaturesCollectionTimeout(
 }
 
 func (s *service) banForfeitCollectionTimeout(
-	ctx context.Context,
-	roundId string,
+	ctx context.Context, roundId string,
 ) {
-	unsignedVtxoKeys := s.cache.ForfeitTxs().GetUnsignedInputs()
+	unsignedVtxoKeys, err := s.cache.ForfeitTxs().GetUnsignedInputs(ctx)
+	if err != nil {
+		log.WithError(err).Warn("failed to get unsigned inputs from cache")
```

Comment:
```suggestion
		log.WithError(err).Warn("failed to get unsigned inputs from cache")
		return
```
we don't want to launch GetVtxos call on empty list ?